README.md

```diff
@@ -11,3 +11,44 @@ A collection of framework specific Auth utilities for working with Supabase.
 
 - [Remix](https://github.com/supabase-community/supabase-auth-helpers/issues/57)
 - [SvelteKit](https://github.com/supabase-community/supabase-auth-helpers/issues/54)
+
+### Apps and Packages
+
+- `nextjs`: a [Next.js](https://nextjs.org) app
```

Comment:
Is this just used to define the term of `nextjs`?

Reply:
No, it's used to tell what kind of apps are in the `examples` directory and the `packages` directory. Maybe we should give these actual names.

Reply:
Then maybe call it "Packages & Examples" and make a nested list?

- Packages
  - ...
- Examples
  - [Next.js](./examples/nextjs)